packages/app/vite.config.ts

```diff
@@ -1,10 +1,9 @@
 import { makeConfig } from '../frontend-shared/vite.config'
 import Layouts from 'vite-plugin-vue-layouts'
 import Pages from 'vite-plugin-pages'
-import Copy from 'rollup-plugin-copy'
```

Comment:
Thank you for that !! 
Great clean-up.

Should we also remove the plugin from the dependencies in `package.json`?

Reply:
Makes sense, I went ahead and removed it.